packages/ComponentLibrary/src/components/Table/customExpandButton.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { MRT_Row } from 'material-react-table';
+import IconButton from '../IconButton';
+import ChevronDownIcon from '../../assets/icons/chevron-down.svg';
+import ChevronUpIcon from '../../assets/icons/chevron-up.svg';
+import ChevronRightIcon from '../../assets/icons/chevron-right.svg';
+import { Organization } from './types';
```

Comment:
This interface must not be included in the main component library folder. If it is to mock a test, create a folder for this use.